modMcf/src/org/aion/mcf/config/CfgPrune.java

```diff
@@ -25,7 +25,13 @@
  *
  * Contributors to the aion source files in decreasing order of code volume:
  *     Aion foundation.
- ******************************************************************************/
+ *     <ether.camp> team through the ethereumJ library.
+ *     Ether.Camp Inc. (US) team through Ethereum Harmony.
+ *     John Tromp through the Equihash solver.
+ *     Samuel Neves through the BLAKE2 implementation.
+ *     Zcash project team.
+ *     Bitcoinj team.
+ */
```

Comment:
This class should have the short license version.

Reply:
nice catch, ty!